endpoints/getting-started-grpc/api_config.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Google Inc.
+# Copyright 2018 Google Inc.
```

Comment:
I think we are supposed to leave these at the original dates since they aren't new files.

Everything else LGTM.